src/bindings/python/flux/job.py

```diff
@@ -267,6 +267,64 @@ def wait(flux_handle, jobid=lib.FLUX_JOBID_ANY):
     return future.get_status()
 
 
+def kill_async(flux_handle, jobid, signum=None):
+    """Send a signal to a running job asynchronously
+
+    :param flux_handle: handle for Flux broker from flux.Flux()
+    :type flux_handle: Flux
+    :param jobid: the job ID of the job to kill
+    :param signum: signal to send (default SIGTERM)
+    :returns: a Future
+    :rtype: Future
+    """
+    if not signum:
+        import signal
```

Comment:
I recommend moving this import up to the top of the file.

Reply:
Ack! Sorry, darnit you already called me out on that one.